documentdb/dub.json

```diff
@@ -4,9 +4,32 @@
 	"copyright": "Copyright © 2015, Adam Wilson",
 	"authors": ["Adam Wilson"],
 	"license": "BSL-1.0",
-	"targetType": "library",
-	"dependencies": {
+	"configurations" : [
+        {
+            "name"           : "documentdb",
+            "targetType"     : "library",
```

Comment:
It got moved under configurations - I found many examples of this and it should be fine.  I believe I tested a normal build with dub.